CDTDatastore/HTTP/CDTHTTPInterceptorContext.m

```diff
@@ -19,16 +19,18 @@
 
 @implementation CDTHTTPInterceptorContext
 
+NSMutableDictionary<NSString*, NSObject*> *_state;
```

Comment:
Not sure what you mean @rhyshort - obviously "normal" properties can't be used for things which are meant to be private, is this something else? An example would be helpful.